index.bs

```diff
@@ -553,7 +553,9 @@ When this method is invoked, the user agent MUST execute the following algorithm
 
 1. For each |authenticator| currently available on this platform, if
     <code>|options|.{{MakeCredentialOptions/attachment}}</code> is [=present|not present=] or its value matches
-    |authenticator|'s attachment modality, [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|.
+    |authenticator|'s attachment modality, [=set/append=] |authenticator| to |currentlyAvailableAuthenticators|. If 
+    |currentlyAvailableAuthenticators| is a empty set, return a {{DOMException}} whose name is "{{NotFoundError}}", and
```

Comment:
This sentence..
> If |currentlyAvailableAuthenticators| is a empty set, return a {{DOMException}} whose name is "{{NotFoundError}}", and terminate this algorithm.

..needs to be a completely separate step, I'm thinking. Otherwise a developer might evaluate it for each authenticator, rather than after all "currently available authnrs", and if the first authenticator is not added to |currentlyAvailableAuthenticators|, then the algorithm is terminated. 

also, it seems to me that..
> If |currentlyAvailableAuthenticators| is a empty set...

..ought to be..
> If |currentlyAvailableAuthenticators| is [=list is empty|empty=]...

..?